docs/en/install-upgrade/installing-stack-demo-self.asciidoc

```diff
@@ -0,0 +1,713 @@
+
+. Run the `elasticsearch-create-enrollment-token` command with the `-s kibana` option to generate a {kibana} enrollment token:
++
+[source,"shell"]
+----
+sudo /usr/share/elasticsearch/bin/elasticsearch-create-enrollment-token -s kibana
+----
+
+. Copy the generated enrollment token from the command output.
+
+. Run the following two commands to enable {kib} to run as a service using `systemd`, enabling {kib} to start automatically when the host system reboots.
++
+["source","sh",subs="attributes"]
+----
+sudo systemctl daemon-reload
+sudo systemctl enable kibana.service
+----
+
+. Before starting the {kib} service there's one configuration change to make, to set {kib} to run on the {es} host IP address. This is done by updating the settings in the `kibana.yml` file. For details about all available settings refer to {kibana-ref}/settings.html[Configure {kib}].
+
+. In a terminal, run the `ifconfig` command and copy the value for the host inet IP address.
+
+. Open the {kib} configuration file for editing:
++
+["source","sh",subs="attributes"]
+----
+sudo vim /etc/kibana/kibana.yml
+----
+
+. Uncomment the line `#server.host: localhost` and replace the default address with the inet value that you copied from the `ifconfig` command. For example:
++
+[source,"yaml"]
+----
+server.host: 10.128.0.28
+----
+
+. Save your changes and close the editor.
+
+. Start the {kib} service:
++
+["source","sh",subs="attributes"]
+----
+sudo systemctl start kibana.service
+----
++
+If you need to, you can stop the service by running `sudo systemctl stop kibana.service`.
+
+. Run the `status` command to get details about the {kib} service.
++
+["source","sh",subs="attributes"]
+----
+sudo systemctl status kibana
+----
+
+. In the `status` command output, a URL is shown with:
+** A host address to access {kib}
+** A six digit verification code
++
+For example:
++
+["source","sh",subs="attributes"]
+----
+Kibana has not been configured.
+Go to http://10.128.0.28:5601/?code=<code> to get started.
+----
++
+Make a note of the verification code.
+
+. Open a web browser to the external IP address of the {kib} host machine, for example: `http://<kibana-host-address>:5601`.
++
+It can take a minute or two for {kib} to start up, so refresh the page if you don't see a prompt right away.
+
+. When {kib} starts you're prompted to provide an enrollment token. Paste in the {kib} enrollment token that you generated earlier.
+
+. Click **Configure Elastic**.
+
+. If you're prompted to provide a verification code, copy and paste in the six digit code that was returned by the `status` command. Then, wait for the setup to complete.
+
+. When you see the **Welcome to Elastic** page, provide the `elastic` as the username and provide the password that you copied in Step 1, from the `install` command output when you set up your first {es} node.
+
+. Click **Log in**.
+
+. On the **Start by adding integrations** prompt, select **Explore on my own**.
```

Comment:
I think we can remove this step. I didn't get this prompt when installing and browsing Kibana this time around. Don't know if it's removed in the latest version or what.